src/Redux/Selectors/ActionsSelectors.js

```diff
@@ -0,0 +1 @@
+export const orderedActionsArraySelector = state => state.Actions.orderedActionsArray;
```

Comment:
is there a reason not to export as default?

Reply:
might be another selectors in the future.
if it doesn't necessary for now we can put it as default.